Towny/src/main/java/com/palmergames/bukkit/towny/listeners/TownyInventoryListener.java

```diff
@@ -22,12 +28,15 @@
 import com.palmergames.bukkit.towny.object.TownyInventory;
 import com.palmergames.bukkit.towny.object.Translatable;
 import com.palmergames.bukkit.util.Colors;
+
 import org.bukkit.inventory.meta.ItemMeta;
 
 import java.util.Set;
 
 public class TownyInventoryListener implements Listener {
 	
+	private static final Component SAVE = Component.text("Save", Style.style().color(TextColor.color(NamedTextColor.GREEN.asHSV())).decorate(TextDecoration.BOLD).build());
+	private static final Component BACK = Component.text("Back", Style.style().color(TextColor.color(NamedTextColor.RED.asHSV())).decorate(TextDecoration.BOLD).build());

```

Comment:
Comparing item names is not exactly ideal anymore, it's better to replace these fields with a NamespacedKey like towny:edit_gui_save and use the PersistentDataContainer to give the item a boolean with that key. Then in the listener you just check if the PDC contains that key rather than checking the item name.